src/trace/listener.ts

```diff
@@ -29,6 +28,12 @@ export interface TraceConfig {
   /**
    * Whether to capture the lambda payload and response in Datadog.
    */
+  captureLambdaPayloadMaxDepth: number;
+  /**
+   * The captured AWS Lambda payloads will become tags of the `aws.lambda` span. This sets how deep
+   * it fathoms the JSON structure. When the max depth reached, the tag's value will be the
+   * stringified value of the deeper nested items.
+   */
```

Comment:
I think you want to put the comment above the variable. Reads to me like the doc comments for capture payload and capture max depth are switched.

Reply:
oops, i certainly messed up the order. good catch!